build.gradle.kts

```diff
@@ -154,7 +159,7 @@ subprojects {
 
     plugins.withId("digital.wup.android-maven-publish") {
         //todo remove afterEvaluate if possible
-        afterEvaluate {
+        beforeEvaluate {
```

Comment:
I had a problem before I changed it.
@dsvoronin I need your opinion. Let discuss it by voice.